app/assets/javascripts/discourse/app/lib/notification-types/linked-consolidated.js

```diff
@@ -0,0 +1,15 @@
+import NotificationTypeBase from "discourse/lib/notification-types/base";
+import { userPath } from "discourse/lib/url";
+import I18n from "discourse-i18n";
+
+export default class extends NotificationTypeBase {
+  get linkHref() {
+    return userPath(`${this.currentUser.username}/notifications`);
```

Comment:
Maybe add a note here explaining that this link is sub-optimal, pending a new page for "linked' in the profile.